src/doc/trpl/SUMMARY.md

```diff
@@ -16,6 +16,7 @@
     * [Iterators](iterators.md)
     * [Concurrency](concurrency.md)
     * [Error Handling](error-handling.md)
+    * [Wrapper Types](choosing-guarantees.md)
```

Comment:
titles matching the URL helps with SEO, so this filename should be 'wrapper-types', to match its title